README.md

```diff
@@ -1,2 +1,29 @@
 # aws-modules-private-ca
 AWS Private CA enables creation of private certificate authority (CA) hierarchies, including root and subordinate CAs
+
+# Documentation
+<!-- BEGIN_TF_DOCS -->
```

Comment:
Could you please use what they use here:
https://github.com/antonbabenko/pre-commit-terraform#terraform_docs?

I don't think the BEGIN_TF_DOCS works with pre-commit?

Reply:
As we discussed this would mean we don't need pre commit hook